src/extension/_locales/en/messages.json

```diff
@@ -66,6 +66,12 @@
   "bulk_result_preview_single_error_no_xlsx": {
     "message": "Microsoft Excel document, convert to Google Sheets first"
   },
+  "bulk_result_preview_single_error_not_supported": {
+    "message": "File type not supported."
+  },
+  "bulk_result_preview_single_error_too_large": {
+    "message": "File too large."
+  },
```

Comment:
This is unrelated to the original issue of throttling, but it helps improve error handling.